docs/Translate.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: default
+title: "The Translate Component"
+---
+
+# `<Translate>`
+
+If you need to translate messages in your own components, React-admin provides the `<Translate>` component which displays your translated messages.
+
+# Usage
+
+```tsx
+const MyHelloButton = () => <button><Translate i18nKey="custom.hello_world" /></button>;
+
+export default MyHelloButton;
+```
+
+**Tip:** You can directly use the translation function with [the `useTranslate` hook](./useTranslate.md).
+
+## Props
+
+| Prop       | Required | Type        | Default | Description                                                     |
+| ---------- | -------- | ----------- | ------- | --------------------------------------------------------------- |
+| `args`     | Optional | `Object`    | -       | The arguments used for pluralization and interpolation.         |
+| `children` | Optional | `ReactNode` | -       | The default content to display if the translation is not found. |
+| `i18nKey`  | Required | `string`    | -       | The translation key.                                            |
+
+## `args`: Pluralization and Interpolation
```

Comment:
This one is called `options` in `useTranslate`. I think we should use the same name.